packages/api/models/schemas.js

```diff
@@ -695,6 +695,7 @@ module.exports.execution = {
     'arn',
     'name',
     'status',
-    'createdAt'
+    'createdAt',
+    'updatedAt'
```

Comment:
Is this update solely due to the convention in the generateRecord/message conventions?      

In either case this is a breaking change as the superclass model stuff allows users to generate executions outside the constraints of the message, and requires an update to the data model CHANGELOG/CHANGELOG.

Reply:
Let's talk about this in person, I don't follow you.

Reply:
This change was made because the schema was wrong here. Functionally, the `updatedAt` has always been required. It is set in the base model's `create`, `update`, and `batchUpdate` methods.

Reply:
You're entirely right, the base class does implement it.   👍